advisories/github-reviewed/2024/10/GHSA-45pg-36p6-83v9/GHSA-45pg-36p6-83v9.json

```
@@ -1,18 +1,14 @@
 {
   "schema_version": "1.4.0",
   "id": "GHSA-45pg-36p6-83v9",
-  "modified": "2024-11-07T19:23:53Z",
+  "modified": "2024-11-05T16:58:28Z",
   "published": "2024-10-29T15:32:05Z",
   "aliases": [
     "CVE-2024-8309"
   ],
   "summary": "Langchain SQL Injection vulnerability",
   "details": "A vulnerability in the GraphCypherQAChain class of langchain-ai/langchain version 0.2.5 allows for SQL injection through prompt injection. This vulnerability can lead to unauthorized data manipulation, data exfiltration, denial of service (DoS) by deleting all data, breaches in multi-tenant security environments, and data integrity issues. Attackers can create, update, or delete nodes and relationships without proper authorization, extract sensitive data, disrupt services, access data across different tenants, and compromise the integrity of the database.",
   "severity": [
```

this was removed by the tool - should this be added back?
```suggestion
  "severity": [
    {
      "type": "CVSS_V3",
      "score": "CVSS:3.0/AV:L/AC:H/PR:N/UI:N/S:U/C:L/I:L/A:L"
    },
```

---

Don't worry! I understand you didn't intend to remove the `CVSS_V3` value and kept the value. 👍